src/agg_bookend.c

```diff
@@ -462,7 +462,7 @@ bookend_serializefunc(PG_FUNCTION_ARGS)
 	if (my_extra == NULL)
 	{
 		fcinfo->flinfo->fn_extra =
-			MemoryContextAlloc(fcinfo->flinfo->fn_mcxt, sizeof(InternalCmpAggStoreIOState));
+			MemoryContextAllocZero(fcinfo->flinfo->fn_mcxt, sizeof(InternalCmpAggStoreIOState));
```

Comment:
For the record, zero-initializations like this aren't necessarily bugs, and come at a cost. Question is if we should add these here or add to suppression file?

Reply:
This actually was a bug, because I expected a field set to zero in the structure.